plugins/docker/docker_

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+#
+# Plugin to graph docker container and storage.
+#
+# GNU GPL Are Tysland & Philip Gabrielsen
+
+TYPE=$( echo $0 | cut -f2 -d'_' )
```

Comment:
Please take care for proper quoting (here: `echo "$0"`).
`shellcheck` will show you all issues.